python_ta/util/monad.py

```diff
@@ -29,14 +29,17 @@ def __str__(self):
         
     def bind(self, fn):
         return fn(self.value)
+
         
-        
-def failable_map(fn : Callable[[TypeVar('T')], Failable], lst : List[Failable]): 
-    # TODO: allow arbitrary containers like tuples
+def failable_map(fn : Callable[[TypeVar('T')], Failable], itr: Iterable[Failable]):
```

Comment:
Let's tidy up the type signature here: remove the space before the first colon, and add a return type. (Similar to `failable_collect` below.